tests/test_symbolics.py

```diff
@@ -620,8 +620,7 @@ def test_minmax_precision(dtype, expected):
     # Check generated code -- ensure it's using the fp64 versions of min/max,
     # that is fminf/fmaxf
     assert all(i in str(op) for i in expected)
-
-    assert np.all(f.data == 6.0)
+    assert np.allclose(f.data, 6.0, rtol=np.finfo(dtype).eps)
```

Comment:
why? we expect perfect equality here

Reply:
@georgebisbas reported issues on
> gcc 11.4,  Intel(R) Core(TM) i7-7700K CPU , ubuntu 22, latest master

I opened the PR in response. I wonder if it's still an issue? I couldn't reproduce, but could understand it being platform specific